test/commands/UnbanBanCommandTest.ts

```diff
@@ -70,7 +71,7 @@ describe("UnbanBanCommand", () => {
 
         it("should be able to detect servers with ban reasons", async () => {
             const mjolnir = createTestMjolnir();
-            (<any>mjolnir).lists = [{listShortcode: "test"}];
+            (<any>mjolnir).policyListManager.lists = [{listShortcode: "test"}];
```

Comment:
While I think there has been a precedent for exposing these utilities externally from Mjolnir, I think we should at least try keeping them internal to Mjolnir and delegating.